Jenkinsfile

```diff
@@ -1,5 +1,35 @@
 // Define the test routine for different python versions
-def test_python(pythonVersion)
+
+def test_python_basic(pythonVersion)
```

Comment:
Can you share the shell stanzas between your `test_python_*` methods? There seems to be quite a bit of copy and paste.

Reply:
Yeah, sure. I'll follow the same approach as in [CouchBackup](https://github.com/cloudant/couchbackup/blob/master/Jenkinsfile#L16).